.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ before_script:
 
 # ================== templates ================== #
 .base_template: &base_template
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:master
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:david.jones_optimize
```

Comment:
@mstbbs I'm assuming this is ok to merge to master like this and then after live release we can update the master image

Reply:
If the image exists, we cool.